examples/kubernetes-yaml/splunk-otel-collector-gateway.yaml

```diff
@@ -0,0 +1,234 @@
+          #from_attribute: YOUR_EXISTING_TAG_NAMEE
+          #action: upsert
+      # Optional: If you want to add an environment tag
+      # If this option is enabled it must be added to the pipeline section below
+      #attributes/newenvironment:
+        #actions:
+        #- key: environment
+          #value: "YOUR_ENVIRONMENT_NAME"
+          #action: insert
+      # Enabling the memory_limiter is strongly recommended for every pipeline.
+      # Configuration is based on the amount of memory allocated to the collector.
+      # The configuration below assumes 2GB of memory. In general, the ballast
+      # should be set to 1/3 of the collector's memory, the limit should be 90% of
+      # the collector's memory up to 2GB, and the spike should be 25% of the
+      # collector's memory up to 2GB. In addition, the "--mem-ballast-size-mib" CLI
+      # flag must be set to the same value as the "ballast_size_mib". For more
+      # information, see
+      # https://github.com/open-telemetry/opentelemetry-collector/blob/main/processor/memorylimiter/README.md
+      memory_limiter:
+        # TODO: Ensure this is set properly
+        ballast_size_mib: ${SPLUNK_BALLAST_SIZE_MIB}
+        check_interval: 2s
+        # TODO: Ensure this is set properly
+        limit_mib: ${SPLUNK_MEMORY_LIMIT_PERCENTAGE}
+        # TODO: Ensure this is set properly
+        spike_limit_mib: ${SPLUNK_MEMORY_SPIKE_PERCENTAGE}
+    exporters:
+      # Traces
+      sapm:
+        # TODO: Ensure this is set properly
+        access_token: "${SPLUNK_ACCESS_TOKEN}"
+        # TODO: Ensure this is set properly
+        endpoint: "https://ingest.${SPLUNK_REALM}.signalfx.com/v2/trace"
+      # Metrics
+      signalfx:
+        # TODO: Ensure this is set properly
+        access_token: "${SPLUNK_TOKEN}"
+        # TODO: Ensure this is set properly
+        realm: "${SPLUNK_REALM}"
+    service:
+      extensions: [health_check, http_forwarder, zpages]
+      pipelines:
+        traces:
+          receivers: [jaeger, opencensus, otlp, sapm, zipkin]
+          processors: [memory_limiter, batch]
+          exporters: [sapm, signalfx]
+        metrics:
+          receivers: [opencensus, otlp, signalfx, prometheus, hostmetrics]
+          processors: [memory_limiter, batch]
+          exporters: [signalfx]
+        logs:
+          receivers: [signalfx]
+          processors: [memory_limiter, batch]
+          exporters: [signalfx]
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: otel-collector
+  labels:
+    app: opentelemetry
+    component: otel-collector
+spec:
+  ports:
+  - name: health # Default Health Check port.
+    port: 13133
+  - name: sapm # Default endpoint for Jaeger gRPC receiver.
+    port: 14250
+  - name: sapm # Default endpoint for Jaeger Thrift receiver.
+    port: 14268
+  - name: sapm # Default endpoint for OpenCensus receiver.
+    port: 55678
+  - name: sapm # Default endpoint for OpenTelemetry receiver.
+    port: 55680
```

Comment:
from otlp receiver docs: ```default = 0.0.0.0:4317 for grpc protocol, 0.0.0.0:55681 http protocol```

Reply:
yeah, this needs to be updated everywhere...